### PR TITLE
Add gas constants

### PR DESCRIPTION
**Pull request by brad-decker**
First PR breaking up from #10965 - This one just adds gas constants for 21000 and 100000 that are shared throughout the app.

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -1,14 +1,3 @@
...
-    "coverage",
-    "dist",
-    "docs",
-    "lavamoat",
-    "node:console",
-    "node_modules",
-    "patches",
-    "test-artifacts"
-  ]
+  "exclude": ["node:console"]
```
Comment (darkwing) [inline jsconfig.json:2]: Is this supposed to be in this PR?  Seems out of place.

File: test/stub/tx-meta-stub.js [hunk 2]
```diff
@@ -16,7 +17,7 @@ export const txMetaStub = {
       type: TRANSACTION_TYPES.SENT_ETHER,
       txParams: {
         from: '0xf231d46dd78806e1dd93442cf33c7671f8538748',
-        gas: '0x5208',
+        gas: GAS_LIMITS,
```
Comment (darkwing) [inline test/stub/tx-meta-stub.js:20]: I think this needs to be `GAS_LIMITS.SIMPLE`

**Review by darkwing — CHANGES_REQUESTED** [review]
I think I spotted an incorrect variable and an extra modified file, so marking as Request Changes.

**Comment by brad-decker** [comment]
thanks @darkwing, ready for another review.

**Review by darkwing — APPROVED** [review]

**Pull request merged**